docs/tutorials/basic_interface.ipynb

```diff
@@ -0,0 +1,837 @@
+       "                        if (nbb_cells[i].get_text() == nbb_unformatted_code) {\n",
+       "                             nbb_cells[i].set_text(nbb_formatted_code);\n",
+       "                        }\n",
+       "                        break;\n",
+       "                    }\n",
+       "                }\n",
+       "            }, 500);\n",
+       "            "
+      ],
+      "text/plain": [
+       "<IPython.core.display.Javascript object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "policy_params, policy_functions = set_up_policy_environment(2020)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The two objects can be passed on to `compute_taxes_and_transfers` with a number of further inputs to compute outputs for a set of data. `policy_params` is a nested dictionary of parameters grouped by different policy types they capture."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "eink_st\n",
+      "eink_st_abzuege\n",
+      "soli_st\n",
+      "arbeitsl_geld_2\n",
+      "arbeitsl_geld\n",
+      "soz_vers_beitr\n",
+      "unterhalt\n",
+      "abgelt_st\n",
+      "wohngeld\n",
+      "kinderzuschlag\n",
+      "kindergeld\n",
+      "elterngeld\n"
+     ]
+    },
+    {
+     "data": {
+      "application/javascript": [
+       "\n",
+       "            setTimeout(function() {\n",
+       "                var nbb_cell_id = 3;\n",
+       "                var nbb_unformatted_code = \"print(*policy_params.keys(), sep=\\\"\\\\n\\\")\";\n",
+       "                var nbb_formatted_code = \"print(*policy_params.keys(), sep=\\\"\\\\n\\\")\";\n",
+       "                var nbb_cells = Jupyter.notebook.get_cells();\n",
+       "                for (var i = 0; i < nbb_cells.length; ++i) {\n",
+       "                    if (nbb_cells[i].input_prompt_number == nbb_cell_id) {\n",
+       "                        if (nbb_cells[i].get_text() == nbb_unformatted_code) {\n",
+       "                             nbb_cells[i].set_text(nbb_formatted_code);\n",
+       "                        }\n",
+       "                        break;\n",
+       "                    }\n",
+       "                }\n",
+       "            }, 500);\n",
+       "            "
+      ],
+      "text/plain": [
+       "<IPython.core.display.Javascript object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "print(*policy_params.keys(), sep=\"\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These keys can be used to extract the exact parametrizations for a given policy group. The example below for instance shows the parameters that concern social security."
```

Comment:
Unfortunately I am not aware of such a place, unless we link to the yaml files directly: https://github.com/iza-institute-of-labor-economics/gettsim/tree/master/gettsim/data ? Do you think this would make sense?

Reply:
Ok, probably not. Then we can just leave it us is right now.